lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,6 +8,18 @@ def define
 
       config = Calculators::BenefitCapCalculatorConfiguration.new
 
+      # Routing question
+      multiple_choice :choose_cap_to_calculate? do
```

Comment:
The text of this question is "Which period do you want to check for?". Did you consider giving the question a similar name?